firmware/README.md

```diff
@@ -4,7 +4,9 @@ Install esp-idf
 --------
 The master board is based on a ESP32 module programmed in C++ using the expressif idf SDK https://github.com/espressif/esp-idf
 
-To install the SDK simply follow the instruction form https://docs.espressif.com/projects/esp-idf/en/stable/get-started/.
+To install the SDK:
+* simply follow the instruction form https://docs.espressif.com/projects/esp-idf/en/stable/get-started/.
+* To install the packages for python2, you need to install packages at specific versions before running the installation: `pip install python-socketio==1.5.0 Flask-SocketIO==2.9`
```

Comment:
Maybe add this into a
```
**Note** : Using **Python2**, you need to install packages at specific versions before running the installation: `pip install python-socketio==1.5.0 Flask-SocketIO==2.9`
```
This might clearer that this is a specific case.

Reply:
Good point. I've added the **Note** block.